baseplate/clients/cassandra.py

```diff
@@ -21,12 +22,38 @@
 from cassandra.query import BoundStatement  # pylint: disable=no-name-in-module
 from cassandra.query import PreparedStatement  # pylint: disable=no-name-in-module
 from cassandra.query import SimpleStatement  # pylint: disable=no-name-in-module
+from prometheus_client import Counter
+from prometheus_client import Gauge
+from prometheus_client import Histogram
 
 from baseplate import Span
 from baseplate.clients import ContextFactory
 from baseplate.lib import config
+from baseplate.lib.prometheus_metrics import default_latency_buckets
 from baseplate.lib.secrets import SecretsStore
 
+cassandra_labels = [
+    "cassandra_contact_points",
```

Comment:
Discussed in slack and have an update to the spec pending to add this (with this exact name). @bjk-reddit said "if it's the cluster hostname then that's probably okay". This essentially is "address" but uses the Cassandra terminology because it's not a single hostname:port.